moxin-protocol/src/protocol.rs

```diff
@@ -28,9 +28,12 @@ pub struct LoadModelOptions {
     pub prompt_template: Option<String>,
     pub gpu_layers: GPULayers,
     pub use_mlock: bool,
+    // if 0, the backend will use the default value
+    pub n_batch: u32,
```

Comment:
What about using `Option` for those fields? Looks cleaner than relying on zero as the "null value".

Reply:
Update